mlflow/catboost.py

```diff
@@ -226,8 +226,12 @@ def log_model(
     :param pip_requirements: {{ pip_requirements }}
     :param extra_pip_requirements: {{ extra_pip_requirements }}
     :param kwargs: kwargs to pass to `CatBoost.save_model`_ method.
+    :return: A `ModelInfo` namedtuple instance that contains the metadata of the logged model,
+             including: `run_id`, `artifact_path`, `model_uri`, `utc_time_created`, `flavors`,
+             `model_uuid`, `saved_input_example_info`, `signature`, `input_schema`, and
+             `output_schema`.
```

Comment:
Can we document `ModelInfo` somewhere to simplify the param description here?

Reply:
This post (written by the sphinx author) describes how to write a document for `namedtuple`:

https://qiita.com/tk0miya/items/d13781658ed38045a693

Reply:
Currently, the namedtuple class is not captured by sphinx, so rewriting the `__doc__` field does not show up in the page either.
I tried `sphinx-toolbox` pip package to support namedtuple, but it has a bug with python 3.7.3 [`_ForwardRef` does not exist in python 3.7.3 while it's supposed to exist](https://github.com/sphinx-toolbox/sphinx-toolbox/blob/c40be459eabee0df4b4d15bd300d015a6fe58c2d/sphinx_toolbox/more_autodoc/typehints.py#L129).

Reply:
In that case, can we make this a Python class?

Reply:
`class ModelInfo(NamedTuple):` seems to be working. Let me try that.